src/Compilers/CSharp/Portable/SymbolDisplay/SymbolDisplayVisitor.Members.cs

```diff
@@ -559,16 +559,16 @@ void visitFunctionPointerSignature(IMethodSymbol symbol)
                         switch (symbol.CallingConvention)
                         {
                             case SignatureCallingConvention.CDecl:
-                                builder.Add(CreatePart(SymbolDisplayPartKind.ClassName, symbol, "Cdecl"));
+                                builder.Add(CreatePart(SymbolDisplayPartKind.Keyword, symbol, "Cdecl"));
                                 break;
                             case SignatureCallingConvention.StdCall:
-                                builder.Add(CreatePart(SymbolDisplayPartKind.ClassName, symbol, "Stdcall"));
+                                builder.Add(CreatePart(SymbolDisplayPartKind.Keyword, symbol, "Stdcall"));
                                 break;
                             case SignatureCallingConvention.ThisCall:
-                                builder.Add(CreatePart(SymbolDisplayPartKind.ClassName, symbol, "Thiscall"));
+                                builder.Add(CreatePart(SymbolDisplayPartKind.Keyword, symbol, "Thiscall"));
                                 break;
                             case SignatureCallingConvention.FastCall:
-                                builder.Add(CreatePart(SymbolDisplayPartKind.ClassName, symbol, "Fastcall"));
+                                builder.Add(CreatePart(SymbolDisplayPartKind.Keyword, symbol, "Fastcall"));
```

Comment:
Those changes are needed for the Quick Info in IDE to show them as keywords instead of classes.

Reply:
do we need to change this?  why not just make the IDE side match the logic here and classify these as classnames?  i don't get why we feel so strongly these need to be keywords.

Reply:
@CyrusNajmabadi It was discussed on Discord and @333fred was concerned about classifying as class names when the compiler isn't actually using the class.

Reply:
I feel like ClassName would be a strange choice. These are contextual keywords, right?

Reply:
@JoeRobich They're not contextual keywords per se. They're "special identifiers" that I can't think of anything similar to them in the language. There is no `SyntaxKind.CdeclKeyword` for example.

They're "hard-coded" in the compiler:

https://github.com/dotnet/roslyn/blob/7ba92d6c1caa1096c7626f72f51b0f619b675218/src/Compilers/CSharp/Portable/Symbols/FunctionPointers/FunctionPointerMethodSymbol.cs#L147-L154

To an end-user, they make look like contextual keywords since it's very similar. In fact, I *think* they could have been implemented as contextual keywords (but it's going to be a bit more complex implementation-wise for nothing).